### DISCUSSION TITLE
Report rate limit

### DISCUSSION THREAD

**Discussion by StormyIceLeopard** [NamelessMC/Nameless#2137] [Ideas]
Copied from here as requested: [https://github.com/NamelessMC/Nameless-Plugin/issues/84](https://github.com/NamelessMC/Nameless-Plugin/issues/84)

My initial request:
> This feature could really help prevent abuse/spam of /report.
>
>I would like to see a cooldown feature for when players report players.
>Like if Alex does /report Steve Fly he is not able to do it again, or at least for a duration of time.
>But still allows Alex to do /report Notch XRAY
>
>Basically not a cooldown on the command itself, but rather a cooldown on the one being reported.

Response:
>Wouldn't a global cooldown be better? By reporting different users every time players can still spam moderators with reports. There can be legitimate reasons for reporting a player multiple times, for example if you see them breaking more rules. Or maybe both: a cooldown for reports in general, and a longer cooldown for reporting a specific user.
>
>In any case, please create an issue for the website instead of the plugin. This should be handled by the website, so reports on the website respect the same cooldown as the plugin. When the plugin submits a report the website can send an error response if the user can't make a report at that time

**Comment by samerton** [comment]
Not sure a website cooldown makes sense for reports, what if there are a wave of spam posts and a user wants to report them all?

Also what if multiple people are breaking rules on the server? Is the cooldown just for reporting the same person?

**Comment by StormyIceLeopard** [comment]
Just reporting the same person.
Kind of how Hypixel /watchdogreport works
